crates/bevy_app/src/app_builder.rs

```diff
@@ -403,6 +445,10 @@ impl AppBuilder {
         self
     }
 
+    /// Initialize a non-send resource in the current [`App`], if it does not exist yet.
+    ///
+    /// Adds a resource that implements `Default` or `FromResources` trait.
+    /// If the resource already exists, it does nothing.
```

Comment:
The last sentence here just repeats the end of the first sentence.
I think it can be removed.